book/P1C5_eha.qmd

```diff
@@ -12,159 +12,456 @@ abstract: TODO (150-200 WORDS)
+Subject `id=46` on the other hand starts in state 0 and only enters state 1 (and thus the risk set for the transitions $1\rightarrow 0$ and $1\rightarrow 2$) after 415 days (`tstart = 415`).
+Other subjects in the data may never enter the risk set for these transitions by remaining in state 0 until the end of follow up or by directly transitioning to state 2.
+The fact that subjects enter the risk sets for different transitions at different time points technically constitutes left-truncation and thus should be taken into account accordingly (@sec-ms-aalen-johanson).
+
+| id| from| to| trans| tstart| tstop| status|treatment  |
+|--:|----:|--:|-----:|------:|-----:|------:|:----------|
+|  1|    1|  0|     3|      0|   151|      0|Placebo    |
+|  1|    1|  2|     4|      0|   151|      1|Placebo    |
+|  8|    1|  0|     3|      0|   211|      1|Prednisone |
+|  8|    1|  2|     4|      0|   211|      0|Prednisone |
+|  8|    0|  1|     1|    211|  2770|      0|Prednisone |
+|  8|    0|  2|     2|    211|  2770|      0|Prednisone |
+| 46|    0|  1|     1|      0|   415|      1|Prednisone |
+| 46|    0|  2|     2|      0|   415|      0|Prednisone |
+| 46|    1|  0|     3|    415|   417|      0|Prednisone |
+| 46|    1|  2|     4|    415|   417|      1|Prednisone |
+: Subset of the `prothr` dataset [@pkgmstate]. {#tbl-ms-data-prothr}
+
+
+### Notation and Definitions
+
+Similar to competing risks (@sec-competing-risks), transitions between different states (see @fig-multi-state) are characterized by transition-specific hazards, denoted by $h_{\ell\rightarrow e}(\tau)$ or short $h_{\ell e}(\tau)$, where $\ell \in \mathcal{L}$ is the starting state and $e \in \mathcal{E}$ the end state, with $\mathcal{L}$ the set of possible starting states (transient states) and $\mathcal{E}$ the set of possible end states (transient and terminal states).
+Technically $e$ should depend on $\ell$, $e(\ell)$, as not all endpoints can be reached from a given starting state.
+For example, in the right panel of @fig-multi-state $\ell \in \{0, 1, 2\}$ and $e(0) \in \{1, 3\}$, $e(1) \in \{0, 2, 3\}$ and $e(2) = \{1, 3\}$.
+However, for notational convenience the difference between $\mathcal{L}$ and $\mathcal{E}$ as well as the dependency of $e$ on $\ell$ is often ignored  and we simply write $\ell, e \in \mathcal{E}$, where $\mathcal{E}= \{0,\ldots, q\}$ includes all possible states.
+
+To formalise the multi-state setting further, let $E(\tau)\in \{0,\ldots,q\}$ the state occupied at time $\tau$.
+Then, the transition specific hazard can be defined as
+<!--  -->
+$$
+\begin{aligned}
+h_{\ell e}(\tau) &=  \lim_{\triangle \tau \to 0} \frac{P(\tau \leq Y < \tau + \triangle\tau, E(\tau + \triangle\tau)=e|Y \geq \tau, E(\tau-) = \ell)}{\triangle \tau}&\nonumber\\
```

Comment:
Is E the random variable for e ? It's a little confusing because it's *similar* to notation for CR but then you redefine e \in \scriptE as the end state without explicitly saying CR is also an end state (though heavily implied above). So I'd probably just recap that E is the RV for e, although TBH I'm actually now confused again because E(tau) hasn't been defined (though I can guess its the event at time tau, in which case though don't you need to define e(tau)?)